README.md

```diff
@@ -44,6 +44,8 @@ Linux    | RPM     | x86          | 0.9.8 | omi-1.1.0.ssl_098.x86.rpm
 
 [Releases]: https://github.com/Microsoft/omi/releases
 
+Alternatively, you can now also download from Microsoft's Azure/Linux Repo. Instructions
+on setting this up can be found [here](https://technet.microsoft.com/en-us/windows-server-docs/compute/Linux-Package-Repository-for-Microsoft-Software).  Follow the instructions for your platform.  You can then install OMI simply with "sudo apt-get install omi".
```

Comment:
"sudo apt install omi" or "sudo yum install omi" depending on platform. Or break then out as 'On Ubuntu, you can use blah, and on CentOS, use blah'. Something like that.